ethereum/llama/aave_treasury/aave_tokens.sql

```diff
@@ -0,0 +1,15 @@
+CREATE SCHEMA IF NOT EXISTS llama;
+
+CREATE TABLE IF NOT EXISTS llama.aave_tokens (   
```

Comment:
Is this table the same as this view? https://github.com/duneanalytics/abstractions/blob/master/ethereum/aave/view_atokens.sql

Reply:
similar, but that one is static, and misses some context on each token. This should update for every live token with additional context (i.e. borrow v deposit, ARC v normal)